source/js/RecipeCard.js

```diff
@@ -0,0 +1,350 @@
+class RecipeCard extends HTMLElement {
+  constructor() {
+    // Part 1 Expose - TODO
+
+    // You'll want to attach the shadow DOM here
+    super();
+    let shadow = this.attachShadow({mode: 'open'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
    let shadow = this.attachShadow({mode: "open"});
```